apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -35,26 +37,35 @@ const PublishNotes: React.FunctionComponent = () => {
     <div className={styles.verticalFlexContainer}>
       {isVisible(visibility) && (
         <div className={styles.customizationContainer}>
-          {MODEL_CARD_FIELDS_LABELS_ICONS.map(([property, label]) => {
+          {MODEL_CARD_FIELDS_LABELS_ICONS.map(([property, label, _]) => {
             const InputTag = getInputTag(property);
+
             return (
               <div className={styles.inputContainer} key={property}>
                 <label htmlFor={property}>
                   <StrongText>{label}</StrongText>
                 </label>
-                <InputTag
-                  id={property}
-                  disabled={isDisabled(visibility)}
-                  value={modelCardInfo[property]}
-                  onChange={event =>
-                    dispatch(
-                      setModelCardProperty({
-                        property: property,
-                        value: event.target.value,
-                      })
-                    )
-                  }
-                />
+                {property === 'exampleTopics' && (
```

Comment:
We've got a lot of these key strings floating around, I think we should figure out a way to pull them from a constant.

Reply:
Yep that could be good, though we do already get automatic checking so long as `property` is typed correctly (which I think here is `keyof ModelCardInfo`). If the key is misspelled/not a key of `ModelCardInfo`, then TS will complain, effectively enforcing that we only compare against strings that are actually keys of `ModelCardInfo` (ex: `"This comparison appears to be unintentional because the types 'keyof ModelCardInfo' and '"hello"' have no overlap.ts(2367)"`)

Reply:
Oh nice, good to know!